plugins-root/check_icmp.c

```diff
@@ -1514,18 +1534,19 @@ get_threshold(char *str, threshold *th)
 }
 
 unsigned short
-icmp_checksum(unsigned short *p, int n)
+icmp_checksum(uint16_t *p, size_t n)
 {
 	unsigned short cksum;
 	long sum = 0;
 
-	while(n > 2) {
-		sum += *p++;
-		n -= sizeof(unsigned short);
+	/* sizeof(uint16_t) == 2 */
```

Comment:
what's the reason of using hard coded 2 instead of calling sizeof?

Reply:
No specific reason, it's the same anyway and at list it is consistent with line 1545 now. Would it be better understandable, if I used `sizeof`?

Reply:
I'd rather change the other occurance as well. But that's only my opinion.